pkg/core/resource/api.go

```diff
@@ -18,6 +22,27 @@ type api struct {
 	client client.Client
 }
 
+func (i Info) EndpointURL(ctx context.Context, op types.Operation, options types.Options) (*url.URL, error) {
+	u, err := url.ParseRequestURI(pathPrefix)
+
+	switch op {
+	case types.OperationCreate, types.OperationDestroy, types.OperationUpdate:
+		return nil, genericAPI.ErrOperationNotSupported
+	}
+
+	if op == types.OperationList {
+		query := u.Query()
+		if len(i.Tags) > 1 {
```

Comment:
technically having this if inside the if below would save some CPU cycles when no tags are given, but indentation may make code a bit less readable ... I'm fine with this, just making sure this is deliberate and not by oversight

Reply:
I always try to avoid nesting when possilbe unless it makes a significant difference. So if it doesn't bother you I'd like to keep it that way. In order to reduce [cognitive complexity](https://docs.codeclimate.com/docs/cognitive-complexity)

Reply:
as said: "I'm fine with this, just making sure this is deliberate and not by oversight" ^^